include/umf/memspace.h

```diff
@@ -44,6 +44,12 @@ umfMemoryProviderCreateFromMemspace(umf_memspace_handle_t hMemspace,
                                     umf_memspace_policy_handle_t hPolicy,
                                     umf_memory_provider_handle_t *hProvider);
 
+///
+/// \brief Retrieves predefined default memspace.
+/// \return default memspace handle on success or NULL on failure.
+///
+umf_memspace_handle_t umfMemspaceDefaultGet(void);
```

Comment:
An alternative API could be a single entry function which accept some enum to specify which predefined memspace to return. For example:
```
enum predefined_memspaces {
    DEFAULT,
    ...
};

umf_memspace_handle_t umfPredefinedMemspaceGet(enum predefined_memspaces) {
    ...
}
```

@igchor, @bratpiorka, @pbalcer what do you think?

Reply:
Yes, this is also possible. I don't really have a preference here.

Reply:
Currently, we use umfXXXPool/ProviderOps() for pools and providers. We should be consistent with that or change this everywhere. But please note that the enum is more problematic when you want to build this list at cmake configuration (enable/disable some compontents).

Reply:
> Currently, we use umfXXXPool/ProviderOps() for pools and providers. We should be consistent with that or change this everywhere.

I do not think the same rules apply everywhere.

> But please note that the enum is more problematic when you want to build this list at cmake configuration (enable/disable some components).

Ok, it makes sense if we want to configure the available predefined memspaces at build-time.